package.json

```diff
@@ -14,14 +19,9 @@
   ],
   "license": "Apache-2.0",
   "repository": {
-    "type": "git",
-    "url": "git+https://github.com/acuvity/acuvity-ts.git"
+    "url": "git+https://github.com/acuvity/acuvity-ts.git",
+    "type": "git"
   },
-  "bugs": {
-    "url": "https://github.com/acuvity/acuvity-ts/issues"
-  },
-  "description": "The official Acuvity TypeScript SDK",
-  "homepage": "https://acuvity.ai/",
```

Comment:
it's pretty annoying that the Speakeasy SDK is flipping this around all the time without any real changes. I'll start a conversation on this with them.